plugins/source/gcp/client/client.go

```diff
@@ -30,9 +36,12 @@ type Client struct {
 
 	ClientOptions []option.ClientOption
 	CallOptions   []gax.CallOption
-	// this is set by table client multiplexer
+
+	EnabledServices map[string]map[string]bool
```

Comment:
I would use `struct{}` here instead of `bool`, sets are simpler to understand with struct{} as it can only have one value.

Reply:
Not sure I follow...

Reply:
The suggestion was to use `map[string]map[string]struct{}` so that it's clear that the `bool` value (of `false`) isn't used (so it's a set, rather than a map of "important" values). Not really a deal breaker but nice to have (it doesn't hinder readability that much)